src/lib/ndpi_utils.c

```diff
@@ -1506,6 +1506,79 @@ int ndpi_dpi2json(struct ndpi_detection_module_struct *ndpi_struct,
 
 /* ********************************** */
 
+char *ndpi_get_ip_proto_name(u_int16_t ip_proto, char *name, int name_len) {
```

Comment:
Since this is a public API function; What do you think about a `name_len` check to make sure it is `> 0` ?

Reply:
Good idea! Done